docs/FAQ.md

```diff
@@ -27,13 +27,13 @@
 
 **Q: Can I create my own UI5 Web Components?**
 
-**A:** Not yet, the framework-level APIs for creating web components are not public yet.
+**A:** Yes, for more information click [here](./dev/Creating%20UI5%20Web%20Components%20Packages.md)
```

Comment:
**A:** Yes, for more information, see [Creating a custom UI5 Web Components package](./dev/Creating%20UI5%20Web%20Components%20Packages.md).